### DISCUSSION TITLE
WQ Human Revamp (Faction Affinities)

### DISCUSSION THREAD

**Discussion by witold-cyrek** [smrealms/smr#946] [Ideas]
The purpose of this post is to let everyone know the ideas I have on table (as in currently working with) as far as revamping WQ Humans.  WQ Humans should be a very interesting SMR Faction, mysterious and hardy, working outside the boundaries of normal Human space  and flaunting all the Galactic Norms.  Sadly, this faction has become something of a joke. No one takes it seriously, and this needs to change.

I have some very cool ideas that I would like to implement for WQ Humans.  And because after these cool things are in, they will be sooooo cool, everyone else will cry about the other factions, I have decided to slowly add some cool stuff to all the races :) All with the use of new Faction Affinity System.  These Affinities will be attached to a Faction in a given game (Allowing for games with no Affinities, or mis-matched affinities), and all players created to this faction will automatically recieve the Affinity.  Affinities can give great bonuses but also bestow penalties, requiring a player to be more reourceful in how they play their character.

**WQ Humans**
**Faction Affinity: Outlaw**

- This Faction is not a memeber of The Federation and operates outside of normally established galactic norms
- This Faction is a member of The Underground
- Ports belonging to this faction openly flaunt Galactic Law, vessels trading there will never be searched for Contraband
- Members of this Faction do not require evil relations in order to trade in Contraband
- Good Aligned members of this faction are considered in bad standing, and thus cannot become President, Visit the Faction HQ, nor Collect Military Payments
- Faction HQ of this faction allows visitors to become Gang Members, lowering their alignment to evil instantly
- Faction HQ allows for collection of Underground Bounties
- Operates in Shadows: Members of this Faction have their Cloak Level increased by 3(*) levels

**Faction Ship Changes**

- New Ship Class "Smuggler" replaces some "Trader" ships
- Smuggler vessels have extra cargo holds built directly into the ship's structure that can be used to transport only Contraband (+25%* trade potential for Contraband)
- Smuggler vessels recieve bonus to evasion of inspections, much like Evil vessels

**Cloak Changes**

These changes will apply to all cloaked ships, but are released with the WQ Humans revamp
- Cloaked vessels appear as Protected/neutral 1 tile away on Local Map
- Cloaked vessels appear as Protected/neutral In Sector unless the player veiwng can see through the cloak.
- Cloaked vessels appear as Enemy In Sector and on Local Map if a player can see through the cloak.

(*) actual numbers not final

**Comment by stupidnewbie** [comment]
The cloak changes can stand on their own.  I'm not sure they're the best way to address the unpopularity of evil and WQ play but they may prove interesting.

WQ trade ships are these weird compromises between hunters, traders, and support ships which end up being an 8 of all trades.  If you flat gave a blockade runner about 40% more holds it would be a reasonable ship to take.  If you double or triple the holds a Negotiator has it would be worth using.  Giving them a smuggler bonus is of no value because red players still need to trade standard goods to make routes function.

Adding a faction affinity to balance a race that has lost its purpose is just another layer of complexity.  What WQ needs is a playable strategy outside of early DM which is currently being made terrible by three factors: lack of people to hunt, PR's making cash near infinite inside of a week, and a 2 week hold on capital ships.  They're just 4 useless ships and a raider that can't play the game it was designed for.
